pkg/registry/flowcontrol/bootstrap/default-objects.go

```diff
@@ -159,6 +159,15 @@ func DefaultFlowSchemas() []*rmv1a1.FlowSchema {
 				Rule:     nonResourceRule([]string{rmv1a1.VerbAll}, []string{rmv1a1.NonResourceAll}),
 			},
 		),
```

Comment:
the code change below does not seem to relate to this PR (that adds NameAll). mind to split it out?

Reply:
done